### PR TITLE
VZ-4947: Add fluentd parsing for Prometheus and reconfig-loader log files

### PR DESCRIPTION
**Pull request by bobdonat**
# Description

This pull request adds fluentd parsing for Prometheus and reconfig-loader log files.  In addition, updated ATs to include testing of both Prometheus and config-reloader.

# Checklist 

As the author of this PR, I have:

- [ ] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml [hunk 1]
```diff
@@ -454,6 +454,61 @@ data:
...
+    <filter kubernetes.**vmi-system-prometheus**verrazzano-system_prometheus**>
+      @type parser
+      @id vmi-system-prometheus
+      key_name log
+      reserve_data true
+      emit_invalid_record_to_error true
+      <parse>
+        # Prometheus has two formats for log records
+        # One with a level and msg
+        # One with a level but not message
```
Comment (davisgomes) [inline platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml:467]: Should this say: "One with a level but not msg"?

**Review by kminder — DISMISSED** [review]
Approved.

File: platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml (hunk 1 above)
Comment (bobdonat) [reply platform-operator/helm_config/charts/verrazzano/templates/verrazzano-logging.yaml]: fixed

**Review by davisgomes — DISMISSED** [review]

**Review by kminder — APPROVED** [review]
Approved.

**Review by kminder — APPROVED** [review]
Approved.

**Pull request merged**